cpp/cmake/thirdparty/get_cccl.cmake

```diff
@@ -23,15 +23,14 @@ function(find_and_configure_cccl)
 
   # Make sure we install cccl into the `include/libcudf` subdirectory instead of the default
   include(GNUInstallDirs)
-  set(CMAKE_INSTALL_INCLUDEDIR "${CMAKE_INSTALL_INCLUDEDIR}/libcudf")
-  set(CMAKE_INSTALL_LIBDIR "${CMAKE_INSTALL_INCLUDEDIR}/lib")
+  set(CMAKE_INSTALL_LIBDIR "${CMAKE_INSTALL_INCLUDEDIR}/libcudf/lib")
```

Comment:
These shouldn't be necessary and indicate the root issue is in rapids-cmake/cpm/cccl.cmake

Reply:
Root fix at: https://github.com/rapidsai/rapids-cmake/pull/513

Reply:
Updated this PR accordingly.